pandas/core/arraylike.py

```diff
@@ -274,8 +274,13 @@ def reconstruct(result):
         result = getattr(ufunc, method)(*inputs, **kwargs)
     else:
         # ufunc(dataframe)
-        mgr = inputs[0]._mgr
-        result = mgr.apply(getattr(ufunc, method))
+        if method == "__call__":
```

Comment:
why is this not checking `__array_ufunc__` ? can you add some comments on what method is above. looking at this method it is way too and minimally documented. pls add some clarity. consider splitting this up (as I had previously asked).

Reply:
> why is this not checking `__array_ufunc__`

The `method` is a keyword from the numpy ufunc protocol, and `"__call__"`  indicates the ufunc is simply called instead of one of its attribute methods (eg `np.maximum(..)` instead of `np.maximum.accumulate(..)` or `np.maximum.reduce(..)`)

Will add some comments